webapp/packages/plugin-connection-custom/src/DriverSelector/DriverSelectorDialog.tsx

```diff
@@ -29,25 +29,18 @@ export const DriverSelectorDialog: DialogComponent<Payload> = observer(function
   const dbDriverResource = useResource(DriverSelectorDialog, DBDriverResource, CachedMapAllKey);
 
   const enabledDrivers = dbDriverResource.resource.enabledDrivers;
-  const dialog = useDriverSelectorDialog(
-    enabledDrivers.map(driver => driver.id),
-    rejectDialog,
-  );
+  const dialog = useDriverSelectorDialog({
+    drivers: enabledDrivers.map(driver => driver.id),
```

Comment:
drivers name can make confusion because "why not to use `useResource` in dialog instead?